src/main/webapp/resources/js/pages/launch/launch-context.js

```diff
@@ -126,6 +146,7 @@ function LaunchProvider({ children }) {
   }
 
   function dispatchLaunch(values) {
+    console.log(values);
```

Comment:
Remove logging statement

Reply:
I want to keep this here for now as it is going to be replaced in the final PR for the UI portion of this series.  I have it so I always know what is in that object.